pkg/cloud/scope/machine.go

```diff
@@ -211,6 +211,9 @@ func (m *MachineScope) DeleteSecretPrefix() error {
 // GetSecretCount returns the number of AWS Secret Manager entries making up
 // the complete userdata
 func (m *MachineScope) GetSecretCount() int32 {
+	if m.AWSMachine.Spec.CloudInit == nil {
```

Comment:
Are we calling this method when `CloudInit` is nil anywhere?

Reply:
no, i think i forgot what we decided last time. deletin.